### PR TITLE
Card layout defects

### PR DESCRIPTION
**Pull request by angelicawink**

### CODE REVIEW

File: src/app/components/hit-text/hit-text.component.ts [hunk 1]
```diff
@@ -60,4 +60,8 @@ export class HitTextComponent {
     };
     return finalFrequencyString
   }
+
+  public isSrcValid(image){
+    debugger
```
Comment (bzmillerboy) [inline src/app/components/hit-text/hit-text.component.ts]: Should this still be in here?

File: src/index.html [hunk 2]
```diff
@@ -12,10 +12,11 @@
   <link rel="manifest" href="manifest.json">
   <link rel="shortcut icon" href="favicon.png">
 
-  <title>Crossroads Group Search</title>
-<script src="https://crds-components.netlify.com/dist/crds-components.js"></script>
+
+  <title>Search Crossroads.net</title>
```
Comment (bzmillerboy) [inline src/index.html]: This should stay as "Crossroads Group Search"

File: src/app/components/hit-text/hit-text.component.ts (hunk 1 above)
Comment (angelicawink) [reply src/app/components/hit-text/hit-text.component.ts]: no it should not, removed!

File: src/index.html (hunk 2 above)
Comment (angelicawink) [reply src/index.html]: not sure how that changed but I've updated it now

**Pull request merged**